extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -106,10 +106,7 @@ export class AmpSelector extends AMP.BaseElement {
     this.init_();
     if (!this.isDisabled_) {
       this.element.addEventListener('click', this.clickHandler_.bind(this));
-      if (this.kbSelectMode_ != KEYBOARD_SELECT_MODES.NONE) {
```

Comment:
Users who use a keyboard as a primary input device, on or off mobile, should be able to select an amp-selector option regardless of the `keyboard-support-mode` attribute.